internal/v1/oscap.go

```diff
@@ -0,0 +1,109 @@
+package v1
+
+import (
+	"errors"
+)
+
+func OscapProfiles(distribution Distributions) (DistributionProfileResponse, error) {
```

Comment:
This method has a similar issue: One commit adds it, and the following one moves it to a separate file. Once again, this has no effect on the result, but makes the history harder to read.